addons/sprite-properties/userscript.js

```diff
@@ -36,11 +36,22 @@ export default async function ({ addon, console, msg }) {
     }
   }
   // Close properties panel when mouse leaves the entire sprite panel
+  let popoverAbove;
   document.body.addEventListener(
-    "mouseleave",
+    "mousemove",
     (e) => {
-      if (e.target.matches('[class*="sprite-selector_sprite-selector_2KgCX"]')) {
```

Comment:
Isn't a possible fix just adding `&& !directionPopoverCurrentlyOpen()` here?
The sprite properties wouldn't collapse until the user moves the cursor once again to the properties, and then leaves it again. But it wouldn't affect performance.

Reply:
A possible implementation of `directionPopoverCurrentlyOpen` could be checking if `e.target.closest('[class*="Popover-above"]')` exists.

Reply:
See #6090